In [3]:
from scipy import stats
from scipy.stats import norm

Pop mean =270 days Sample mean = 260 days Sample SD = 90 days Sample n = 18 bulbs df = n-1 = 17

In [ ]:
# Assume Null Hypothesis is: Ho = Avg life of Bulb >= 260 days
# Alternate Hypothesis is: Ha = Avg life of Bulb < 260 days

In [4]:
# find t-scores at x=260; t=(s_mean-P_mean)/(s_SD/sqrt(n))
t=(260-270)/(90/18**0.5)
t

-0.4714045207910317

Let's find P(X>=260) for null hypothesis

In [5]:
# p_value=1-stats.t.cdf(abs(t_scores),df=n-1)... Using cdf function
p_value=1-stats.t.cdf(abs(-0.4714),df=17)
p_value

0.32167411684460556

In [6]:
#  OR p_value=stats.t.sf(abs(t_score),df=n-1)... Using sf function
p_value=stats.t.sf(abs(-0.4714),df=17)
p_value

0.32167411684460556

Therefore, probability that 18 randomly selected bulbs would have an average life of no more than 260 days is 32.17%.Assuming significance value α = 0.05 .Thus, as p-value > α ; Accept Ho i.e. The CEO claims are false and the avg life of bulb > 260 days